rts/motoko-rts/src/memory/ic.rs

```diff
@@ -28,9 +28,18 @@ extern "C" {
     pub(crate) fn get_static_roots() -> Value;
 }
 
+pub(crate) unsafe fn get_aligned_heap_base() -> u32 {
+    // align to 32 bytes
+    (get_heap_base() + 31) / 32 * 32
```

Comment:
```suggestion
    ((get_heap_base() + 31) / 32) * 32
```

We also have a "rounding up division" function in Rust core now, but it has a [strange name](https://doc.rust-lang.org/beta/std/primitive.u32.html#method.unstable_div_ceil) for [complicated reasons](https://github.com/rust-lang/rust/issues/88971).

Reply:
That name is indeed ugly. Will remember this, and might change when available regularly.